### DISCUSSION TITLE
LOTW Sync Questions

### DISCUSSION THREAD

**Discussion by minorsecond** [magicbug/Cloudlog#1317] [Q&A] [answered]
I have two questions regarding LoTW Syncing:

1. Is syncing supposed to be done automatically? Currently I'm having to manually sync it, and I don't think automatic syncing is happening.
2. There is an entry under LoTW Matches table that shows up every time I manually sync. Is there a reason this would be happening? The info is below.

KD5LPB | 2021-10-28 21:37 | AD0DA | FT8 | Y | 2021-10-29 00:00 | UT | QSO Record: Found | LoTW Record: Updated

**Comment by magicbug** [comment, marked as answer]
1. LoTW will automatically sync if you have correctly setup the corn jobs as per the install guide as per https://github.com/magicbug/Cloudlog/wiki/Recommended-Cron-Jobs

2. Cloudlog asks LoTW for changes since last download, could be that for some reason that one keeps being returned by them.
